src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/listeners/HMPPSDomainEvent.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.prisonertonomisupdate.listeners
+
+import uk.gov.justice.digital.hmpps.prisonertonomisupdate.incentives.IncentivesService
+
+data class HMPPSDomainEvent(
+  val eventType: String,
+  val prisonerId: String? = null,
+  val additionalInformation: IncentivesService.AdditionalInformation? = null,
+)
```

Comment:
moved out to its own file as it is now shared

Reply:
might be nice to have domain event definition per subject to avoid lots of nullables eventually